### ISSUE TITLE
Only Java files are being generated

### ISSUE THREAD

**Issue by ivanmonteiro** [jhipster/jhipster-dotnetcore#23]
Tested commit 90d3406975a4116da5d192bc71ab0595e139168f and it generates C# code.
But with the latest commits jhipster only creates java files. Seems that the dotnetcore generator is not called. Others are having related issues.
I'm using Windows 10, Node v10.16.0 LTS and jHipster v6.1.2

> > @danielpetisme Thanks a lot, was able to "npm link" inside this repository cloned to my local disk (on MacOS). Then I created a folder for my app and went in there (cd ), did "npm link generator-jhipster-dotnetcore" which created a folder "node_modules" inside. After running "jhipster -d --blueprint dotnetcore" I see that the blueprint is picked up (as evidenced in launch sequence of jHipster: "INFO! Options: d: true, blueprint: dotnetcore, from-cli: true") but it generates Java application nevertheless. Is there some questionnaire answer that I choose that screws things up? (I chose monolithic with Postgres, no distributed cache, with Angular)
> > Note: I have "dotnet" available as part of 2.2.300 SDK installed on my MacOS.
> 
> same errors. it is still generating java files.
> Please update readMe with more details

Having the same issues, jhipster generates only java files

_Originally posted by @ivanmonteiro in https://github.com/jhipster/jhipster-dotnetcore/issues/2#issuecomment-512996099_

**Comment by danielpetisme** [comment]
Can you provide a reproducer please.
I'm on a Mac and I can't reproduce the issue but a friend who is using Windows share the same concern.
We need more details to troubleshoot.

@BananeGuimauve you're using Windows too? Did you face the issue?

**Comment by BananeGuimauve** [comment]
I just pull it and tested it on my personal computer at my home and it works too. Both computers are on Windows 10.
I will explain you a reproducer : 
1) I go into my blueprint folder
2) I use "npm link" command
3) Now I create a new folder for example on my Desktop where I will generate the app and I go into
4) I use "npm link generator-jhipster-dotnetcore" command
5) Now, I obtain a node_modules folder containing this file tree to the generator-jhipster needles :
generator-jhipster-dotnetcore\node_modules\generator-jhipster\generators\client\needle-api/needle-client*.js
That's the path I use to modify the needles in the 9057f6218cb5458845941fe4c4eb8ec04444d59c commit in the file "generators/client/index.js". The lines are, for each needle modified : 
- 96, 108
- 120, 132
- 144, 156
- 168,184
6) After, I call the blueprint with "jhipster -d --blueprint dotnetcore" command and add entities with "jhipster -d --blueprint dotnetcore entity bankAccount" for example and it works.

**Comment by ivanmonteiro** [comment]
> I just pull it and tested it on my personal computer at my home and it works too. Both computers are on Windows 10.
> I will explain you a reproducer :
> 
> 1. I go into my blueprint folder
> 2. I use "npm link" command
> 3. Now I create a new folder for example on my Desktop where I will generate the app and I go into
> 4. I use "npm link generator-jhipster-dotnetcore" command
> 5. Now, I obtain a node_modules folder containing this file tree to the generator-jhipster needles :
>    generator-jhipster-dotnetcore\node_modules\generator-jhipster\generators\client\needle-api/needle-client*.js
>    That's the path I use to modify the needles in the [9057f62](https://github.com/jhipster/jhipster-dotnetcore/commit/9057f6218cb5458845941fe4c4eb8ec04444d59c) commit in the file "generators/client/index.js". The lines are, for each needle modified :
> 
> * 96, 108
> * 120, 132
> * 144, 156
> * 168,184
> 
> 1. After, I call the blueprint with "jhipster -d --blueprint dotnetcore" command and add entities with "jhipster -d --blueprint dotnetcore entity bankAccount" for example and it works.

With commit 90d3406 everything is working fine. But with more recent commits only Java files are generated. My guess is that this blueprint is NOT being called.
I used the method of [Running local Blueprint version for development](https://github.com/jhipster/jhipster-dotnetcore#running-local-blueprint-version-for-development) to test it on Windows 10, Node v10.16.0 LTS and jHipster v6.1.2
I will try the install method using npm install with the github url and report here soon.

**Comment by BananeGuimauve** [comment]
I managed to reproduce your problem by cloning the project and make a new "npm link". 
Thanks @ivanmonteiro, thanks to you I found the commit and the line causing the problem.
In the commit 539208903c08c85520a53a03f6efa78452e0d8a2 a needed dependancy "to-pascal-case": "^1.0.0" in package.json has been deleted. I think @danielpetisme and me didn't noticed it because the dependancy was already installed localy and that didn't cause the crash of the blueprint. It's only with your issue and by creating a new clone of the project and a new installation than we encountered the problem. 
Thank you, I will make a PR closing the issue now. :)
